vip-cache-manager/vip-cache-manager.php

```diff
@@ -27,15 +27,6 @@ static public function instance() {
 	}
 
 	public function __construct() {
-		// Execute the healthcheck as quickly as possible
```

Comment:
Should we add a note that the health check endpoint is defined somewhere else?

Reply:
Either way - I probably wouldn't be looking in this class for the healthcheck endpoint first (it wouldn't be expected to me that it's in here). We know it was in here b/c historical reasons, but I dunno if future us or others would be expecting to see it here.

Reply:
The reason I think I’d look here is I know the health checks are in varnish.